src/local/fs-helper.js

```diff
@@ -45,7 +45,13 @@ export class FSHelper extends EventEmitter {
     let fileContents = null;
 
     if (this._paused || util.isExcluded(sourcepath, this._excludes)) return;
-    if (evt === 'add' || evt === 'change') fileContents = fs.readFileSync(sourcepath).toString();
+    if (evt === 'add' || evt === 'change') {
+      try {
+        fileContents = fs.readFileSync(sourcepath).toString();
+      } catch (err) {
+
```

Comment:
Might be good to do some sort of logging when there's a failure... I'm sure a user would want to know why things are out of sync